sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -82,7 +82,7 @@ class OrcFileFormat
   with DataSourceRegister
   with Serializable {
 
-  override def shortName(): String = "orc"
+  override def shortName(): String = "orcV1"
```

Comment:
Is there any other file-based data source instance to use 'V1' postfix?

Reply:
No, but if the FileFormat implementation has the same shortName as V2, then there can be a problem in data source lookup. Although Orc this won't be an issue since there is some branch for selecting native Orc data source.

Reply:
Then, let's keep the original~ Like you, I also tried renaming when I split `native`/`hive` ORC implementation. The feedback was negative.  I believe DSv2 migration plan is not intended to touch this of DSv1.

Reply:
I can understand your concern. I think we had better find a solution for such naming problems. It is not only about ORC, but the other data source migrations will also face the same problem.

Reply:
Of course, +1 for that. We need a clear rule (and maybe document).

Reply:
Talked to @cloud-fan offline just now. Let's keep the short name unchanged. For other data sources, we can resolve v1/v2 sources in `DataSource.lookupDataSource`.

Reply:
Thanks!